Assets/Scripts/Town/PlayerMovementComponent.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using CustomInput;
+
+public class PlayerMovementComponent : MonoBehaviour {
+    public float movementSpeed;
```

Comment:
Move to settings mentioned before (or after idk how github will place it). `TownSettings` probably.